etc/calc_release_version_selftest.sh

```diff
@@ -67,7 +67,7 @@ echo "Test next minor version ... begin"
     # failed, then it is probably because a new major/minor release was made.
     # Update the expected output to represent the correct next version.
     # XXX NOTE XXX NOTE XXX
-    assert_eq "$got" "3.10.0-$DATE+git$CURRENT_SHORTREF"
+    assert_eq "$got" "4.1.0-$DATE+git$CURRENT_SHORTREF"
```

Comment:
Noticed when validating [this commit](https://github.com/mongodb/mongo-cxx-driver/pull/1381/commits/2aff0993b0033a3a327d46139b4b4df60e900bf4) in https://github.com/mongodb/mongo-cxx-driver/pull/1381.